packages/one-app-dev-bundler/package.json

```diff
@@ -44,7 +44,7 @@
     "postcss-modules": "^3.2.2",
     "read-pkg-up": "^9.0.0",
     "regenerator-runtime": "^0.13.9",
-    "sass": "^1.54.8",
+    "sass": "1.65.1",
```

Comment:
Why does the version of `sass` need to be pinned to this specific version?

Reply:
great spot, I had issues running OneApp locally because Sass recently updated to be exclusively ESM and it mucked up the way that consumers need to handle imports. This file shouldn't have been committed though, it was just changed locally to bump myself past their bad release

Reply:
[cc6b6d8](https://github.com/americanexpress/one-app-cli/pull/557/commits/cc6b6d8e89cc5fc2db508b63763c23f2e3c902d7)